opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -992,6 +992,10 @@ type Provider {
   strategy: String
   provider: String
 }
+type UserStatus {
+  status: String!
```

Comment:
I wonder if it would be better if status is an enum. What do you think?

Reply:
@Archidoit - they were an ENUM on the initial submission but in this commit @richard-julien swapped the behavior. (https://github.com/OpenCTI-Platform/opencti/pull/3303/commits/ccd802daa29109878c3438cb4c2d08c88700a275)

Reply:
okay, let's keep a string then